### PR TITLE
Add documentation on integration with Confluent

### PR DESCRIPTION
**Pull request by liuzix**
### What is changed, added or deleted? (Required)

- How to create an Avro changefeed.
- A quick start guide for integrating with Confluent Platform.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/quick-start-confluent.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
```
Comment (amyangfei) [inline ticdc/quick-start-confluent.md:1]: will this doc be added to toc

**Review by leoppro — APPROVED** [review]
LGTM

File: ticdc/confluent-integration-using-ticdc.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+title: Quick Start Guide on Integrating TiDB with Confluent Platform
```
Comment (TomShawn) [inline ticdc/confluent-integration-using-ticdc.md:2]: @kissmydb Please help determine where this document should be placed in TOC. Thanks~

File: ticdc/confluent-integration-using-ticdc.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    confluent local services connect connector list
+    ```
+
+## Integration procedures
+
+1. Add the following configuration into `jdbc-sink-connector.json`:
```
Comment (TomShawn) [inline ticdc/confluent-integration-using-ticdc.md:32]:
```suggestion
1. Save the following configuration into `jdbc-sink-connector.json`:
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -206,6 +206,21 @@ The following are descriptions of parameters and parameter values that can be co
...
+Sample configuration:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+--sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&protocol=avro&partition-num=6&max-message-bytes=67108864&replication-factor=1"
+--opts registry="http://127.0.0.1:8081"
+```
+
+To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to other sinks such as Elasticseach and Cassandra, you should use `avro`protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**. Do **NOT** use them in the production environment.
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:220]:
```suggestion
To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to other sinks such as Elasticseach and Cassandra, you should use `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**. Do **NOT** use them in the production environment.
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -206,6 +206,21 @@ The following are descriptions of parameters and parameter values that can be co
...
+Sample configuration:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+--sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&protocol=avro&partition-num=6&max-message-bytes=67108864&replication-factor=1"
+--opts registry="http://127.0.0.1:8081"
+```
+
+To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to other sinks such as Elasticseach and Cassandra, you should use `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**. Do **NOT** use them in the production environment.
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:220]:
```suggestion
To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to other sinks such as Elasticseach and Cassandra, you should use `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**. Do **NOT** use them in a production environment.
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
+---
+title: Quick Start Guide on Integrating TiDB with Confluent Platform
```
Comment (lilin90) [inline ticdc/integrate-confluent-using-ticdc.md:2]:
```suggestion
title: Quick Start Guide on Integrating TiDB with the Confluent Platform
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+summary: Learn how to stream TiDB data to the Confluent Platform using TiCDC.
+---
+
+# Quick Start Guide on Integrating TiDB with Confluent Platform
```
Comment (lilin90) [inline ticdc/integrate-confluent-using-ticdc.md:6]:
```suggestion
# Quick Start Guide on Integrating TiDB with the Confluent Platform
```

File: ticdc/manage-ticdc.md [hunk 8]
```diff
@@ -206,6 +206,25 @@ The following are descriptions of parameters and parameter values that can be co
...
+{{< copyable "shell-regular" >}}
+
+```shell
+--sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&protocol=avro&partition-num=6&max-message-bytes=67108864&replication-factor=1"
+--opts registry="http://127.0.0.1:8081"
+```
+
+To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to other sinks such as Elasticseach and Cassandra, you should use `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**.
+
+For detailed integration guide, see [Quick Start Guide on Integrating TiDB with Confluent Platform](/ticdc/integrate-confluent-using-ticdc.md).
```
Comment (lilin90) [inline ticdc/manage-ticdc.md:226]:
```suggestion
For the detailed integration guide, see [Quick Start Guide on Integrating TiDB with the Confluent Platform](/ticdc/integrate-confluent-using-ticdc.md).
```

File: ticdc/manage-ticdc.md [hunk 9]
```diff
@@ -206,6 +206,25 @@ The following are descriptions of parameters and parameter values that can be co
...
+Sample configuration:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+--sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&protocol=avro&partition-num=6&max-message-bytes=67108864&replication-factor=1"
+--opts registry="http://127.0.0.1:8081"
+```
+
+To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to other sinks such as Elasticseach and Cassandra, you should use `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**.
```
Comment (lilin90) [inline ticdc/manage-ticdc.md:224]:
```suggestion
To use the [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) provided by Confluent to stream data to other sinks such as Elasticsearch and Cassandra, you should use the `avro` protocol and provide a URL for [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) in `opts`. Note that the `avro` protocol and Confluent integration are **experimental**.
```

**Review by amyangfei — COMMENTED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**